compiler/src/main/java/org/robovm/compiler/AppCompiler.java

```diff
@@ -196,6 +203,19 @@ public Config getConfig() {
             classes.add(clazz);
         }
         
+        if (!config.getCustomIBClasses().isEmpty()) {
+            for (String s : config.getCustomIBClasses()) {
+                if (s == null || s.trim().isEmpty()) {
```

Comment:
You probably want to ensure that any custom class names added to `Config` are not null not empty before adding them to the `Config`. Config is kinda supposed to be immutable (it's not really anymore). But at least we should try not to return nulls or empty strings when not necessary. Do the check in line 390 and only add if the class name is != null and !empty.